### ISSUE TITLE
Sizes for iPhone 6(S) Plus and iPad Pro

### ISSUE THREAD

**Issue by jrunning** [audreyr/favicon-cheat-sheet#56]
The [iOS Human Interface Guidelines](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/IconMatrix.html#//apple_ref/doc/uid/TP40006556-CH27-SW2) now include sizes for the iPhone 6(S) and iPad Pro. Here's the lot of them:

* iPhone 6(S) Plus (@3x): 180x180
* iPhone 4S, 5, 6, 6S (@2x): 120x120
* iPad, iPad mini (@2x): 152x152
* iPad 2, iPad mini (@1x): 76x76
* iPad Pro (@2x): 167x167

**Comment by ChrisChiera** [comment]
Was about to add a note about the missing 180x180 for iPhone 6 as well.

**Comment by bgandon** [comment]
+1
Btw, many recent PRs are pending. Is there still anyone maintaining this project?

**Comment by pydanny** [comment]
Yes, project is still maintained. However, there has been a bit of work overload and medical emergency in the way.

**Comment by bgandon** [comment]
Ok, no pb, thanks for the reply!

**Comment by audreyfeldroy** [comment]
Added! iPad Pro 167x167 and iPhone 6 Plus 180x180 are now in the document.
